src/main/resources/db/migration/postgresql/R__6410_correspondents_json_by_case.sql

```diff
@@ -0,0 +1,27 @@
+SET search_path TO casework;
```

Comment:
I'm not sure why this needs to be changed, it doesn't use completed. We shouldn't need this migration at all.